### PR TITLE
Implement logging joins and ability to greet members

### PR DESCRIPTION
**Pull request by ghost**
Self-explanatory, impl of #56

### CODE REVIEW

**Comment by ghost** [comment]
Awaiting review from @VelvetThePanda

File: src/Silk.Core/Tools/EventHelpers/IConfigService.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿namespace Silk.Core.Tools.EventHelpers
+{
+    internal interface IConfigService
```
Comment (VelvetToroyashi) [inline src/Silk.Core/Tools/EventHelpers/IConfigService.cs]: What's this doing? Is this what's being injected into the services? There should be IConfigService in Core/Services/Interfaces

Comment (VelvetToroyashi) [reply src/Silk.Core/Tools/EventHelpers/IConfigService.cs]: Correction: There isn't.

**Review by VelvetToroyashi — APPROVED** [review]
Looks good 👍🏽

**Pull request merged**